cicecore/cicedyn/general/ice_flux.F90

```diff
@@ -566,6 +588,9 @@ subroutine alloc_flux
          fcondtopn_f(nx_block,ny_block,ncat,max_blocks), & ! downward cond flux at top surface (W m-2)
          fsensn_f   (nx_block,ny_block,ncat,max_blocks), & ! sensible heat flux (W m-2)
          flatn_f    (nx_block,ny_block,ncat,max_blocks), & ! latent heat flux (W m-2)
+         evapn_f    (nx_block,ny_block,ncat,max_blocks), & ! evaporative water flux (kg/m^2/s) by atmosphere model
+         dflatndTs_f(nx_block,ny_block,ncat,max_blocks), & ! derivative of flatn with respect to Ts
+         dfsurfndTs_f(nx_block,ny_block,ncat,max_blocks), & ! derivative of fsurfn with respect to Ts
```

Comment:
If `Ts` is surface temperature, please use `Tsfc` or else `surface temperature` or `sfc temp`. (`s` could also indicate snow, etc)